### PR TITLE
AAF and "no SRP" mode added to Simple Listener example

### PR DESCRIPTION
**Pull request by ghMichalW**
I would like to propose a change that allow to run simple listener in static configuration without SRP. Additionally I would like to add AAF option for additonal packet format support. 

The change is dictated by a fact that Automotive AVB that does not support stream protocol reservation.

### CODE REVIEW

**Comment by ghMichalW** [comment]
I have done the final checks and from my side the pull request is ready to merge.

**Comment by andrew-elder** [comment]
These changes look ok to me. Anyone else?

File: examples/simple_listener/simple_listener.c [hunk 1]
```diff
@@ -170,90 +235,130 @@ void sigint_handler(int signum)
...
+			break;
+		case 'i':
+			dev = strdup(optarg);
+			break;
+		case 'f':
+			file_name = strdup(optarg);
+			break;
+		case 'n':
+			glob_no_srp = 1;
+			sprintf(dest_mac,"%s", static_dest_mac);
```
Comment (bdthomsen) [inline examples/simple_listener/simple_listener.c]: Not sure why sprintf was used here, rather than strcpy(dest_mac, static_dest_mac).  However, it works as is.

Comment (andrew-elder) [inline examples/simple_listener/simple_listener.c]: @ghMichalW - do you have any comments on @bdthomsen's comment?

Comment (pinealservo) [inline examples/simple_listener/simple_listener.c]: I think it's fine, and the pull request has been open long enough. I'd be happy to merge another pull request that changes it to strcpy if someone wants to write it, though.
